feature/platform/fabric/otg_tests/sampled_backplane_capacity_counters_test/README.md

```diff
@@ -32,26 +30,10 @@ the available backplane capacity counters correctly while forwarding traffic
 
```

Comment:
should be ON_CHANGE as this leaf is not expected to change frequently?

Reply:
Yes, updated